### DISCUSSION TITLE
Tips to avoid jumping using quasi-random

### DISCUSSION THREAD

**Discussion by Minh309** [KULeuvenNeuromechanics/PredSim#177] [Q&A] [answered]
Hi,

While running the simulation, sometimes my model got stuck in local minima using quasi-random guess. Do you have any tips to avoid this?

https://github.com/user-attachments/assets/438080ba-91ac-43f0-8bfe-dbec40a885cb

**Comment by VDB-Bram** [comment]
Can you share the settings you used?

You say 'sometimes'; do you have a feeling yourself what might be causing this?

**Reply by Minh309** [reply]
Hi,

I think including the passive torque of the shoulders might be the issues:
Here are the settings I used:
 S.subject.set_damping_coefficient_selected_dofs = {{'arm_flex','arm_add','arm_rot'},1,'elbow_flex',1,'mtp_angle',1};            

And I added the K and theta values to the arm dofs like this: 
K.arm_flex= [-2.44 5.05 1.51 -11.88];
theta.arm_flex = [-0.5 1.81];
K.arm_add= [-15.03 20.94 5.03 -14.93];
theta.arm_add = [-1.57 0.05];
K.arm_rot= [-0.43 14.94 0.43 -14.93];
theta.arm_rot = [-0.15 0.15];
K.elbow_flex= [2.09 -6.94 -11.03 11.33];
theta.elbow_flex = [2.4 0.3];

The values were extracted from Ross Miller's model 
https://simtk.org/projects/umocod

**Reply by Minh309** [reply, marked as answer]
It's confirmed that adding passive torques at the arms caused the issue. By removing those K and theta values of the arm dofs, I can avoid getting stuck at local minima.

**Reply by VDB-Bram** [reply]
Good to know what caused the issue, thanks for sharing. Still wondering though why this happens.
